dist/profile/manifests/kubernetes/resources/evergreen.pp

```diff
@@ -0,0 +1,66 @@
+#   Class: profile::kubernetes::resources::evergreen
+#
+#   This class deploys the Jenkins Essentials backend service layer, a
+#   container which communicates with a provisioned Azure PostgreSQL database
+#
+class profile::kubernetes::resources::evergreen (
+    Array $clusters = $profile::kubernetes::params::clusters,
+    Array $domain_alias = [],
+    String $image_tag = 'latest',
```

Comment:
If you plan to always us 'latest' tag, you'll need a way to tell kubernetes to restart running pods.
A trick would be  to add an information to the deployment resource that change every weeks/days/months
Something like 
metadata:
  date:  $(date "+%V")